src/lib/ndpi_utils.c

```diff
@@ -2177,17 +2184,41 @@ ndpi_http_method ndpi_http_str2method(const char* method, u_int16_t method_len)
   case 'G': return(NDPI_HTTP_METHOD_GET);
   case 'H': return(NDPI_HTTP_METHOD_HEAD);
 
+  case 'L':
+    if (strncmp(method, "LOCK", 4) == 0)
+      return(NDPI_HTTP_METHOD_LOCK);
+    break;
+
+  case 'M':
+    if (strncmp(method, "MOVE", 4) == 0)
+      return(NDPI_HTTP_METHOD_MOVE);
+    else if (strncmp(method, "MKCO", 4) == 0)
+      return(NDPI_HTTP_METHOD_MKCOL);
+    break;
+
```

Comment:
Check (only) for `method[1]` (like the `P` case)

Reply:
> Check (only) for `method[1]` (like the `P` case)

Well, there are a two other HTTP methods that start with MK - `MKWORKSPACE` and `MKACTIVITY`. These methods are defined in RFC 3253 (Versioning Extensions to WebDAV), but I thought I wouldn't add it because this standard is marked as proposed.

Reply:
Hm... All of this shit only relates to WebDAV anyway.